fuzzbench/run_experiment.py

```diff
@@ -18,16 +19,22 @@
 import redis
 import rq
 
+from common import config_utils, environment, yaml_utils
 from fuzzbench import fake_jobs
 
 
-def run_experiment():
+def run_experiment(config):
     """Main experiment logic."""
     print('Initializing the job queue.')
     queue = rq.Queue()
     jobs = []
-    for i in range(6):
-        jobs.append(queue.enqueue(fake_jobs.build_image, 'something-%d' % i))
+    jobs.append(queue.enqueue(fake_jobs.build_image, 'base-images'))
+    for benchmark in config.get('benchmarks'):
+        jobs.append(queue.enqueue(fake_jobs.build_image, benchmark))
+    for fuzzer in config.get('fuzzers'):
+        for benchmark in config.get('benchmarks'):
+            jobs.append(queue.enqueue(fake_jobs.build_image,
+                                      fuzzer + benchmark))
```

Comment:
Are you planning to get https://github.com/google/fuzzbench/pull/548 in first, and then update this part?

Reply:
Yes, probably.

Reply:
SG! Let's get back to this after that's done (you can re-request review then). Thanks!